resources/ts/Components/Integrations/Detail/CredentialsLegacyAuthConsumers.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Heading } from "../../Heading";
+import { useTranslation } from "react-i18next";
+import { Integration } from "../../../Pages/Integrations/Index";
+import { StatusLight } from "../../StatusLight";
+import { CopyText } from "../../CopyText";
+import { ActivationFlow } from "../../ActivationFlow";
+import { Alert } from "../../Alert";
+import { IntegrationStatus } from "../../../types/IntegrationStatus";
+import { Credentials } from "./Credentials";
+
+type Props = {
+  email: string;
+  id: Integration["id"];
+  status: Integration["status"];
+  subscription: Integration["subscription"];
+  type: Integration["type"];
+} & Credentials;
```

Comment:
Same remark
```suggestion
type Props = Pick<Integration, 'email' | 'id' | 'status' | 'subscription' | 'type' > & Credentials;
```

Maybe you can even define it elsewhere since it is used in multiple places, to avoid duplicate code...